test/parallel/test-stream2-unpipe-drain.js

```diff
@@ -55,6 +51,6 @@ src1.once('readable', function() {
 
 
 process.on('exit', function() {
-  assert.equal(src1.reads, 2);
-  assert.equal(src2.reads, 2);
+  common.mustCall(src1._read, 2);
```

Comment:
This is incorrect. `common.mustCall()` should be added prior to the `process.on('exit', ...)` handler.